node/node_test.go

```diff
@@ -244,6 +245,61 @@ func TestRegisterProtocols_OpenRPC_WS(t *testing.T) {
 	}
 }
 
+func TestDiscoverCompliesOpenRPCSpec(t *testing.T) {
+	stack, err := New(testNodeConfig())
+	if err != nil {
+		t.Fatalf("failed to create protocol stack: %v", err)
+	}
+	defer stack.Close()
+
+	datadir := filepath.Join(os.TempDir(), "node-test")
+	defer os.RemoveAll(datadir)
+
+	stack.config = &DefaultConfig
+	stack.config.HTTPHost = DefaultHTTPHost
+	// enable all available APIs
+	stack.config.HTTPModules = []string{"admin", "debug", "eth", "ethash", "miner", "net", "personal", "rpc", "trace", "txpool", "web3"}
+	stack.config.DataDir = datadir
+
+	if err := stack.Start(); err != nil {
+		t.Fatal(err)
+	}
+	defer stack.Close()
+
+	client, err := rpc.Dial("http://localhost:8545")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer client.Close()
+
+	res := make(map[string]interface{})
+	err = client.Call(&res, "rpc_discover")
+	if err != nil {
+		t.Errorf("client call: %v", err)
+	}
+
+	openRPCSpecURL := "https://github.com/open-rpc/meta-schema/releases/download/1.14.0/open-rpc-meta-schema.json"
```

Comment:
@meowsbits do you know of a better URL to use so as we test always with the latest one. Using the master branch didn't worked.

Reply:
I looked at the tagged v1.14.0 version too, same issue I'm guessing: https://github.com/open-rpc/meta-schema/blob/1.14.0/schema.json#L21.

```json
      "enum": ["GENERATED FIELD: Do Not Edit - if you are seeing this unexpectedly, use released meta-schema (https://github.com/open-rpc/meta-schema/releases)"]
```

Looks like `/releases/` is the suggested way.

Reply:
Yup, same issue. I am afraid that we will forget to change it on a newer spec release. :(

Any ideas on how to handle this? I believe that they might have a URL somewhere serving the latest spec always.

Reply:
I think I'd just go with their suggested 
> use released meta-schema (https://github.com/open-rpc/meta-schema/releases)

And would maybe 

```
curl https://github.com/open-rpc/meta-schema/releases/download/1.14.0/open-rpc-meta-schema.json -o ./testdata/open-rpc-meta-schema-v1.14.0.json
```

and use

```go
 	openRPCSpecURL := "./testdata/open-rpc-meta-schema-v1.14.0.json" 
```

Thoughts?

Reply:
I had already done before reading the comment. But based on this I also moved it in a testdata folder.
Solved by https://github.com/etclabscore/core-geth/pull/295/commits/86157cb7fb658bc8c2c1d1d50f45e3ce578fbfdc